src/types/hero.ts

```diff
@@ -0,0 +1,17 @@
+export interface IThumbnail {
+  path: string
+  extension: string
+}
+export type Hero = {
+  id: number
+  description: string
+  name: string
+  thumbnail: IThumbnail
+}
+
+export type Comic = {
```

Comment:
Hero and Comics types are the same, you can use interface inheritance to remove duplication

Reply:
done